crypto/signature.go

```diff
@@ -7,6 +7,11 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/refs"
 )
 
+// MarshalledStably describes structs which can be marshalled transparently.
+type MarshalledStably interface {
```

Comment:
this name for interface type looks weird to me, i'd call it `Marshaler`, don't think there will be another marshalers in the package

Reply:
do not get naming too

Reply:
`crypto.Marshaler` is too generic and we have `StableMarshal()` inside of it, so the thing that has `StableMarshal()` implemented can be marshaled (or marshalled) stably. Hence the type. We can use a little less fancy name for it, but that needs to be something better than `Marshaler`.

Practically though people will just pass some random v2 structures there that will just work.

Reply:
`StablyMarshallable`, yay, that might be a better one.

Reply:
renamed